<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkToFUtilViewControls</class>
 <widget class="QWidget" name="QmitkToFUtilViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>452</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <spacer name="m_Spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>311</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="SurfacePropertiesBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Surface</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="m_SurfaceCheckBox">
        <property name="text">
         <string>Surface</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="m_KinectTextureCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Kinect RGB Texture</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="m_TextureCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Texture</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="m_TriangulationThreshold">
        <property name="toolTip">
         <string>If this value is &gt;0.0, a triangulation will only be applied for vertices within this threshold distance (in mm).</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>If this value is &gt;0.0, a triangulation will only be applied for vertices within this threshold distance (in mm).</string>
        </property>
        <property name="text">
         <string>Triangulation Threshold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QmitkToFRecorderWidget" name="m_ToFRecorderWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QmitkToFVisualisationSettingsWidget" name="m_ToFVisualisationSettingsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QmitkToFCompositeFilterWidget" name="m_ToFCompositeFilterWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QmitkToFPointSetWidget" name="tofMeasurementWidget" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="QmitkToFConnectionWidget" name="m_ToFConnectionWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkToFRecorderWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkToFRecorderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkToFVisualisationSettingsWidget</class>
   <extends>QWidget</extends>
   <header>QmitkToFVisualisationSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkToFCompositeFilterWidget</class>
   <extends>QWidget</extends>
   <header>QmitkToFCompositeFilterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkToFPointSetWidget</class>
   <extends>QWidget</extends>
   <header>QmitkToFPointSetWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkToFConnectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkToFConnectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
